templates/etcd-ha/3/README.md

```diff
@@ -0,0 +1,38 @@
+# Etcd 
+
+A distributed, highly-available key/value store for storing configuration information.
+
+### General
+
+This template attempts to create an N-node Etcd cluster. Only 1 node is allowed per host. If not enough hosts are available, the largest cluster possible will be built with the resources available. Adding more hosts at a later date will result in the cluster scaling up to the maximum desired size.
+
+### Upgrades
+
```

Comment:
Instead of putting in limitations/what's coming in a future release within sections, could we make a whole section at the beginning or end so it' smore clear to users?